Translations of `Operational Studies` component updated

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OSRD/Operational Studies](https://hosted.weblate.org/projects/osrd/operational-studies/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/osrd/operational-studies/horizontal-auto.svg)

gaetan-osrd commented on chart/templates/stateful_editoast/deployment.yaml:
```diff
@@ -13,7 +13,7 @@ metadata:
     {{- toYaml $mergedAnnotations | nindent 4 }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.services.statefulEditoast.replicas | default 2 }}
```
This change seems unrelated to the prupose of the PR

Commit:
front: operational-studies: add translation using weblate (Chinese (Simplified Han script))

Translation: OSRD/Operational Studies
Translate-URL: https://hosted.weblate.org/projects/osrd/operational-studies/zh_Hans/

Signed-off-by: Languages add-on <noreply-addon-languages@weblate.org>

wf16110144 committed:
front: operational-studies: translated using weblate (Chinese (Simplified Han script))

Currently translated at 0.1% (1 of 557 strings)

Translation: OSRD/Operational Studies
Translate-URL: https://hosted.weblate.org/projects/osrd/operational-studies/zh_Hans/

Signed-off-by: 王飞 <1071607129@qq.com>

flomonster committed:
Deleted translation using Weblate (Chinese (Simplified Han script))

flomonster commented:
Empty PR
